src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -93,12 +76,24 @@ bool IsGitToolsMessageVisible()
                 RepoName = ActiveRepoName;
                 RepoUrl = ActiveRepoUri.ToString();
                 Icon = GetIcon(false, true, false);
+
+                var weekElapsed = DateTimeOffset.Now.GetIso8601WeekOfYear() > settings.WelcomeMessageLastSeen.GetIso8601WeekOfYear();
```

Comment:
This won't give you an elapsed week, it will only tell you whether a day is on a different week of the year than another day - i.e., Saturday and Sunday will be on different weeks.

To check whether one day is more than 7 days from another day, do `(DateTimeOffset.Now - settings.WelcomeMessageLastSeen).Days > 6`